src/main/scala/akka/persistence/cassandra/query/EventsByTagPublisher.scala

```diff
@@ -135,6 +143,9 @@ private[query] class EventsByTagPublisher(
 
     case Cancel =>
       context.stop(self)
+
+    case tagWritten: String if tagWritten == tag =>
```

Comment:
would it make sense to have a `case class TagWritten(tag: String)` that's published instead of a raw string? I initially didn't do that because of serialization concerns.